app/src/main/AndroidManifest.xml

```diff
@@ -1,11 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
     package="com.github.braillesystems.learnbraille">
 
     <uses-permission android:name="android.permission.VIBRATE" />
     <application
         android:name="com.github.braillesystems.learnbraille.LearnBrailleApplication"
         android:allowBackup="false"
+        tools:replace="android:allowBackup"
```

Comment:
Why did this line was added?

Reply:
In a newly added library (perfectTune) `android:allowBackup` is set to `true`. This line overrides those settings, otherwise you get an error ["manifest merger failed"](https://stackoverflow.com/questions/46934387/attribute-applicationallowbackup-value-false-from-androidmanifest-xml-is-also)